packages/llvm.rb

```diff
@@ -283,7 +283,8 @@ def self.build
 cxx_sys=#{CREW_PREFIX}/include/c++/#{gcc_ver}
 cxx_inc=#{CREW_PREFIX}/include/c++/#{gcc_ver}/x86_64-cros-linux-gnu
 gnuc_lib=#{CREW_PREFIX}/lib/gcc/x86_64-cros-linux-gnu/#{gcc_ver}
```

Comment:
Better fix? `gnuc_lib=#{CREW_LIB_PREFIX}/gcc/x86_64-cros-linux-gnu/#{gcc_ver}`

Reply:
Yeah. I agree. Otherwise, I suppose this is fine.